packages/app/src/pages/404/index.tsx

```diff
@@ -38,3 +38,17 @@ const NotfoundPage: React.FC<Props> = () => {
 };
 
 export default NotfoundPage;
+
+export const query = graphql`
```

Comment:
Official example did the same thing.
https://github.com/microapps/gatsby-plugin-react-i18next/blob/master/example/src/pages/index.js

Reply:
[As far as I know](https://www.gatsbyjs.com/docs/how-to/querying-data/page-query/#the-longer-answer), graphql tag should be placed in each page.


For now, it would be acceptable to place the code on each page, even if it appears to resemble boilerplate code.